modules/references/client/components/CreateReference.component.js

```diff
@@ -150,10 +156,10 @@ export default function CreateReference({ userFrom, userTo }) {
         onSelect={() => {}}
         id="create-reference-tabs"
       >
-        <Tab eventKey={0} title={t('How do you know them')} disabled>
+        <Tab eventKey={0} title={t(`How do you know ${userToName}`)} disabled>
```

Comment:
Let's leave the username out from here since it makes the tab overly lengthy and doesn't add any value; it's already clear from the page itself.

<img width="1199" alt="image" src="https://user-images.githubusercontent.com/87168/79735190-8f8f6700-8300-11ea-8c34-15c5ccd7a102.png">

Furthermore, the username is secondary identification while their "display name" is the primary identification used thorough the UI otherwise.

Reply:
We might need to shorten this for mobile anyway, in which case short "Interaction" might work best.